### PR TITLE
Split it boy.

### PR DESCRIPTION
**Pull request by vzhufk**
Split it in modules.

### CODE REVIEW

File: src/search.js [hunk 1]
```diff
@@ -1,9 +1,20 @@
 const map = require("./map");
 
-module.exports = (code, comp) => {
-  const list = require(`../assets/${code}`);
-  return list.reduce((a, v) => {
-    v = map(v);
+module.exports = (data, comp) => {
+  let isCompresed = false;
+  if (typeof data === "string") {
+    data = require(`@z1p/${data.toLowerCase()}`);
```
Comment (stepanvanzuriak) [inline src/search.js]: Maybe something like this?
```javascript
    try {
      data = require(`@z1p/${data.toLowerCase()}`);
    } catch (e) {
      console.error(
        `@z1p/${data.toLowerCase()} not installed!
        Please run "npm install @z1p/${data.toLowerCase()}"`
      );
    }
```

File: src/search.js [hunk 2]
```diff
@@ -1,9 +1,20 @@
 const map = require("./map");
 
-module.exports = (code, comp) => {
-  const list = require(`../assets/${code}`);
-  return list.reduce((a, v) => {
-    v = map(v);
+module.exports = (data, comp) => {
+  let isCompresed = false;
```
Comment (stepanvanzuriak) [inline src/search.js]: Typo: `isCompressed`

File: scripts/folders.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+const fs = require("fs");
```
Comment (stepanvanzuriak) [inline scripts/folders.js:1]: Useless `folders.js` file?

File: scripts/data.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+const fs = require("fs");
```
Comment (stepanvanzuriak) [inline scripts/data.js:1]: Looks like `data.js` and `package.js` do almost the same, is it ok?

File: scripts/readme.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+const fs = require("fs");
+
+const iso = require("iso-country-codes");
+
+const CODES = fs.readdirSync("./assets");
+
+CODES.forEach(code => {
+  const name = iso.byAlpha2[code].name;
```
Comment (stepanvanzuriak) [inline scripts/readme.js]: Consider moving into some global file, to not repeat this in `readme.js` and `package.js`

File: src/search.js (hunk 1 above)
Comment (vzhufk) [reply src/search.js]: Whatever. You can apply it.

File: src/search.js (hunk 2 above)
Comment (vzhufk) [reply src/search.js]: I think, that the better way is to remove `isCompressed` at all. And create `.compress` function for  third party datasets.

File: scripts/folders.js (hunk 3 above)
Comment (vzhufk) [reply scripts/folders.js:1]: It would be used. Some day.

File: scripts/data.js (hunk 4 above)
Comment (vzhufk) [reply scripts/data.js:1]: `scripts` folder is in `.npmignore`. So it's for us, in case of changes.

**Review by stepanvanzuriak — APPROVED** [review]

**Review by malyuchok — APPROVED** [review]

**Pull request merged**